1911 Encyclopedia Britannica.js

```diff
@@ -0,0 +1,56 @@
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+
+/*
+1911 Encyclopedia Britannica Translator
+Copyright (C) 2015 Jonathan Meyer
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.-
+*/
+
+
+FW.Scraper({ 
+	itemType : 'encyclopediaArticle',
+	encyclopediaTitle : "1911 Encyclopædia Britannica",
+	detect : FW.Url().match(/(encyclopedia\.jrank\.org\/).{7}(\/).+/),
+	title : FW.Xpath('//html/body/div[1]/div[3]/h1').text(),
+	attachments : [{
+		url : FW.Url(),
+		title : "Jrank Snapshot",
+		type : "text/html"
+		}],
+	abstractNote : FW.Xpath('//html/body/div[1]/blockquote[1]/b').text().trim(),
```

Comment:
OK, I see what you're scraping here and that you're covering this case later. But based on http://encyclopedia.jrank.org/CAL_CAR/CALHOUN_JOHN_CALDWELL_1782_185o.html this is not the sort of information that should be put into abstractNote. You can put it in a note attachment though, if you think this is useful information to maintain.